src/js/control-bar/mute-toggle.js

```diff
@@ -81,11 +81,11 @@ class MuteToggle extends Button {
     const vol = this.player_.volume();
     let level = 3;
 
-    if (vol === 0 || this.player_.muted()) {
+    if (this.player_.muted()) {
```

Comment:
That works for me. Only reason I changed the other levels is because I thought no ring seems like a good indicator for 0 sound, 1 ring for up to 50% and two rings for over 50%